src/workloads/execution/FilterWithComplexLogicalExpression.yml

```diff
@@ -426,6 +434,535 @@ Actors:
               ]
             }
 
+# Clean up the data from the previous phase.
+- Name: ClearCollection
```

Comment:
Couple of questions:
- Would it be better to insert all of the data up front and change the paths in the queries? I'm not sure if it's idiomatically correct to do what I've done here, which is alternate queries with collection drops/inserts.
- Would it be better to move these to another file entirely?